src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,16 @@
-export const UserInfo = () => {};
+import usersFromServer from '../../api/users';
+import { getUser } from '../../helperFunctions/getUser';
+
+interface Props {
+  userId: number;
+}
+
+export const UserInfo: React.FC<Props> = ({ userId }) => {
```

Comment:
```suggestion
export const UserInfo: React.FC<Props> = ({ user }) => {
```
you can take a user arg, cause you already found the user in the parent component